src/bundle/DependencyInjection/Configuration/Parser/SubtreePath.php

```diff
@@ -0,0 +1,73 @@
+<?php
+
+/**
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+declare(strict_types=1);
+
+namespace EzSystems\EzPlatformAdminUiBundle\DependencyInjection\Configuration\Parser;
+
+use eZ\Bundle\EzPublishCoreBundle\DependencyInjection\Configuration\AbstractParser;
+use eZ\Bundle\EzPublishCoreBundle\DependencyInjection\Configuration\SiteAccessAware\ContextualizerInterface;
+use Symfony\Component\Config\Definition\Builder\NodeBuilder;
+
+/**
+ * Configuration parser for subtree path strings.
+ *
+ * Example configuration:
+ * ```yaml
+ * ezpublish:
+ *   system:
+ *      default: # configuration per siteaccess or siteaccess group
+ *          subtree_path:
+ *              content: '/1/2/'
+ *              media: '/1/43/'
+ * ```
+ */
+class SubtreePath extends AbstractParser
+{
+    public function addSemanticConfig(NodeBuilder $nodeBuilder)
+    {
+        $nodeBuilder
+            ->arrayNode('subtree_path')
+                ->info('Subtree paths configuration')
+                ->addDefaultsIfNotSet()
+                ->children()
+                    ->scalarNode('content')
+                        ->info('Subtree path of Content')
```

Comment:
Please provide better description i.e. `Location path for Content Structure top menu item`